<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Skynet Context Injector â€” Memory Loader v1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>ðŸ”¹ SKYNET CONTEXT INJECTOR ðŸ”¹</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="subtitleLabel">
      <property name="text">
       <string>Memory Loader â€” Inject context into AI agents</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="contextFileLayout">
      <item>
       <widget class="QLabel" name="contextFileLabel">
        <property name="text">
         <string>Context File:</string>
        </property>
        <property name="minimumWidth">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="contextFilePathEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>No file selected...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectFileButton">
        <property name="text">
         <string>Select Context File</string>
        </property>
        <property name="minimumWidth">
         <number>150</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="fileInfoLabel">
      <property name="text">
       <string>File info: -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="agentLayout">
      <item>
       <widget class="QLabel" name="agentLabel">
        <property name="text">
         <string>Select Agent:</string>
        </property>
        <property name="minimumWidth">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="agentComboBox">
        <property name="minimumWidth">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="agentInfoLabel">
      <property name="text">
       <string>Agent info: -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="launchButton">
      <property name="text">
       <string>ðŸš€ LAUNCH WITH CONTEXT INJECTION</string>
      </property>
      <property name="minimumHeight">
       <number>50</number>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="logLabel">
      <property name="text">
       <string>Activity Log:</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="logTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Logs will appear here...</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottomLayout">
      <item>
       <widget class="QPushButton" name="clearLogButton">
        <property name="text">
         <string>Clear Log</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="viewHistoryButton">
        <property name="text">
         <string>View Injection History</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Status: Ready</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
